app/controllers/candidate_interface/carry_over_controller.rb

```diff
@@ -10,20 +10,19 @@ def start
     def create
       CarryOverApplication.new(current_application).call
       flash[:success] = 'Your application is ready for editing'
-      redirect_to candidate_interface_before_you_start_path
+      redirect_to candidate_interface_application_form_path
     end
 
   private
 
-    # How will we know if a form has been carried over?
     def redirect_if_already_carried_over
       return if must_be_carried_over?
 
       redirect_to candidate_interface_application_form_path
     end
 
     def must_be_carried_over?
-      current_application.not_submitted_and_deadline_has_passed? || current_application.unsuccessful_and_apply_2_deadline_has_passed?
+      (RecruitmentCycle.current_year >= current_application.recruitment_cycle_year) && current_application.not_submitted_and_deadline_has_passed? || current_application.unsuccessful_and_apply_2_deadline_has_passed?
```

Comment:
Worth adding some parens to clarify the relationship between  `&&` and `||` here?

Reply:
https://github.com/DFE-Digital/apply-for-teacher-training/pull/5159/commits/5b60d75b82b083725067625a90b52e2794b1e940